gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ObservationMapper.java

```diff
@@ -222,9 +233,48 @@ private String getComment(List<RCMRMT030101UKPertinentInformation02> pertinentIn
         zeroSequenceComment.ifPresent(stringJoiner::add);
         postFixedSequenceComments.ifPresent(stringJoiner::add);
 
+        // Append episodicity to the comment.
+        qualifiers.ifPresent(q -> appendEpisodicity(q, stringJoiner));
+
         return stringJoiner.toString();
     }
 
+    /**
+     * Append episodicity to comment separating from existing comments with <br> tag.
+     * @param qualifiers
+     * @param stringJoiner
+     */
+    private void appendEpisodicity(List<CR> qualifiers, StringJoiner stringJoiner) {
+        qualifiers.stream()
+                .map(this::buildEpisodicityText)
+                .filter(Objects::nonNull)
+                .forEach(et -> {
```

Comment:
I don't think you need these extra brackets -> {} in here around "stringJoiner.add...", you can remove them which should improve the clarity of code

Reply:
Thanks @ole4ryb  - I've done this, can you re-review please?